_posts/2023-03-21-tables.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: displaying beatiful tables with Bootstrap Tables
+date: 2023-03-20 14:37:00-0400
+description: an example of how to use Bootstrap Tables
+categories: sample-posts
+giscus_comments: true
+related_posts: true
+datatable: true
+# TODO: fix spacing below table
+# TODO: stylize table a little bit (e.g. header)
```

Comment:
the table styles look good to me. are you planning on making additional changes? if not, let's remove these comments.